### PR TITLE
module two

### PR DESCRIPTION
**Pull request by nikabranickaepam**
Added:

- Implementation for task 1
- Implementation for task 2

### CODE REVIEW

File: tasks/module/two/task_1.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+trash = ['rubbish', 'chewed gum', 'used tissue']
+
+
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+
+    for item_name, item_count in inventory.items():
+        print(f"{str(item_count)} {item_name} ")
```
Comment (Pacu2) [inline tasks/module/two/task_1.py:16]: You don't need `str` here

Comment (nikabranickaepam) [reply tasks/module/two/task_1.py]: Removed here and in other places where it was redundant

File: tasks/module/two/task_1.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+LIGHT_WEIGHT_THRESHOLD = 60
+HEAVY_WEIGHT_THRESHOLD = 70
+MAX_CAPACITY_THRESHOLD = 80
+
+
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+
```
Comment (Pacu2) [inline tasks/module/two/task_1.py:20]: I think this empty line is unnecessary. Usually, we use empty lines to separate visually different parts of given function, `item_total` above is really connected to the for loop below.

File: tasks/module/two/task_1.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    for item_name, item_count in inventory.items():
+        print(f"{item_count} {item_name}")
+        item_total += item_count
+
```
Comment (Pacu2) [inline tasks/module/two/task_1.py:24]: Same with this empty line

File: tasks/module/two/task_1.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
+from collections import defaultdict
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+
+my_inventory = {'gold coin': 42, 'rope': 1}
+
+dragon_loot = ['gold coin', 'chewed gum', 'dagger', 'gold coin', 'gold coin', 'ruby', 'rubbish', 'chewed gum',
+               'used tissue']
+
+trash = ['rubbish', 'chewed gum', 'used tissue']
```
Comment (Pacu2) [inline tasks/module/two/task_1.py:10]: `trash` might not be a proper naming, as you know what they say. One's man trash is another man's treasure.

how about `items_to_skip`?

File: tasks/module/two/task_1.py (hunk 2 above)
Comment (nikabranickaepam) [reply tasks/module/two/task_1.py]: Removed

File: tasks/module/two/task_1.py (hunk 3 above)
Comment (nikabranickaepam) [reply tasks/module/two/task_1.py]: Removed

File: tasks/module/two/task_1.py (hunk 4 above)
Comment (nikabranickaepam) [reply tasks/module/two/task_1.py]: Changed :)

**Pull request merged**